examples/experimental/trial/flexible_primitives_mnist_pytorch/model_def.py

```diff
@@ -0,0 +1,172 @@
+"""
+This example shows how to interact with the Determined PyTorch interface to
+build a basic MNIST network.
```

Comment:
Update this comment to explain a little bit about the flexible primitives project.  Also explain why the names have an underscore, that we expect them to change, and that we will be removing the underscore when the API is finalized.